File: C/calculator.c
#include <stdio.h>
#include <stdlib.h>

int main(void)
{   
    int response;
    int num1;
    int num2;

    printf("Welcome, please enter an operation such as: +,-,*,/ \n");
    response = getchar();

    printf("\nWhat is the first number you want to use? \n");
    scanf(" %d", &num1);
    printf("\nWhat is the second number you want to use? \n");
    scanf(" %d", &num2);

    switch(response){
        case '0':
            break;

        case '+':   // Addition
        printf("\n%i\n",num1+num2);
            break;

        case '-':   // Subtraction
        printf("\n%i\n",num1-num2);
            break;

        case '/':   // Division
        printf("\n%i\n",num1/num2);
            break;
            
        case '*':   // Multiplication
        printf("\n%i\n",num1*num2);
            break;

        default:
        printf("An exception occured, please restart the program. \n");
            break;

    }
    return 0;
}

File: JavaScript/calculator.js
// Yes I know, this isn't in "console" but this was the best I could do with stock JavaScript. 
function calc(){
    var response = window.prompt("Please enter a valid operation (+,-,*,/)");
    var num1 = window.prompt("Please enter the first number");
    var num2 = window.prompt("Please enter the second number");

    first = Number(num1)
    second = Number(num2)
    
    switch(response){
        case '+':
        total = first + second; // Addition
        alert(total);
            break;
    
        case '-':
        total = first - second; // Subtraction
        alert(total);
            break;
    
        case '*':
        total = first * second; // Multiplication
        alert(total);
            break;
    
        case '/':
        total = first / second // Division
        alert(total);
            break;
    }
}
